lib/jsdom/living/events/EventTarget-impl.js

```diff
@@ -197,13 +214,17 @@ function invokeEventListeners(listeners, target, eventImpl) {
     }
 
     const listener = handlers[i];
+    const { capture, once } = listener.options;
```

Comment:
Unfortunately we have not yet done a major version bump to require Node v6, so we cannot yet use destructuring.